.github/workflows/test-e2e.yml

```diff
@@ -15,15 +15,16 @@ jobs:
     name: "Compute test matrix"
     runs-on: ubuntu-latest
     env:
-      GH_HEAD_REF: ${{ github.head_ref }}
+      GH_REF: ${{ github.ref }}
```

Comment:
Shouldn't be better to set this env var in the context of the step? This will prevent someone use it as generally available value for any GH event

Reply:
I simplified even further, moving the expansion to the script directly